docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -497,6 +500,19 @@ iceberg.catalog.type=rest
 iceberg.rest-catalog.uri=http://iceberg-with-rest:8181
 ```
 
+`iceberg.security` must be `read_only` when connecting to Databricks Unity catalog
```

Comment:
What happens when a user forgets ot set `iceberg.security=read_only` ?

Reply:
Write operations will result in the following error: 
```
Query 20241015_222748_00008_srrgj failed: Unable to process: Status: 405
Description: Method Not Allowed
```